a/apache-freemarker/Dockerfiles/ubi_8.5/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM registry.access.redhat.com/ubi8:8.5
+
+ARG PACKAGE_VERSION=v2.3.32
+
+RUN yum install -y sudo \
```

Comment:
Do we need `sudo` ?
As we are not setting the user using the `user` command.

Also, Can we club all the`RUN` commands to single one?
Also, after extracting tars remove them.
That saves some amount of Docker image size.

Reply:
@kandarpamalipeddi made the changes